packages/expect/src/__tests__/asymmetricMatchers.test.ts

```diff
@@ -278,32 +288,40 @@ test('ObjectNotContaining does not match', () => {
       first: objectContaining({second: {}}),
     }).asymmetricMatch({first: {second: {}}}),
     objectNotContaining({}).asymmetricMatch(null),
+    objectNotContaining({}).asymmetricMatch(undefined),
     objectNotContaining({}).asymmetricMatch({}),
   ].forEach(test => {
     jestExpect(test).toEqual(false);
   });
 });
 
 test('ObjectNotContaining inverts ObjectContaining', () => {
-  [
-    [{}, null],
-    [{foo: 'foo'}, {foo: 'foo', jest: 'jest'}],
-    [{foo: 'foo', jest: 'jest'}, {foo: 'foo'}],
-    [{foo: undefined}, {foo: undefined}],
-    [{foo: undefined}, {}],
-    [{first: {second: {}}}, {first: {second: {}}}],
-    [{first: objectContaining({second: {}})}, {first: {second: {}}}],
-    [{first: objectNotContaining({second: {}})}, {first: {second: {}}}],
-    [{}, {foo: undefined}],
-  ].forEach(([sample, received]) => {
+  (
+    [
+      [{}, null],
+      [{foo: 'foo'}, {foo: 'foo', jest: 'jest'}],
+      [{foo: 'foo', jest: 'jest'}, {foo: 'foo'}],
+      [{foo: undefined}, {foo: undefined}],
+      [{foo: undefined}, {}],
+      [{first: {second: {}}}, {first: {second: {}}}],
+      [{first: objectContaining({second: {}})}, {first: {second: {}}}],
+      [{first: objectNotContaining({second: {}})}, {first: {second: {}}}],
+      [{}, {foo: undefined}],
+    ] as const
+  ).forEach(([sample, received]) => {
```

Comment:
Was considering to use `test.each`, but it was hard to invent a title for each case. Without unique titles it did not look good. So perhaps `forEach` is fine in this file?